src/Core/Cache/RedisCacheDriver.php

```diff
@@ -0,0 +1,77 @@
+<?php
```

Comment:
Why is adding a new cache handler in a PR titled with code beautification?

Some github foo?